1

```diff
@@ -0,0 +1,17 @@
+Add test for recurring links and clean up method of retrieving recurring
```

Comment:
@eileenmcnaughton you and your odd ball files

Reply:
frigg - fixing